interfaces/builtin/dm_crypt.go

```diff
@@ -61,6 +61,9 @@ mount options=(rw,nosuid,nodev) /dev/dm-[0-9]* -> /{,run/}media/**,
 
 # mount/umount (via libmount) track some mount info in these files
 /run/mount/utab* wrlk,
+
+# Allow access to the file locking mechanism
+/run/cryptsetup rwk,
```

Comment:
Should this also include `/var/run/` as well as the more standard `/run` prefix? I notice we generally do this for other locations under `/run`. ie should it instead be:

```
/{,var/}run/cryptsetup rwk,
```

Also does this require access to any files within `/run/cryptsetup` or only just to the directory itself?

Finally, if we do change this to be `/{,var/}run/cryptsetup rwk,` then line 63 should also be changed to add the `{,var/}` prefix too.

Reply:
`/run/cryptsetup` is just a file, @mardy any feedback from your perspective?

Reply:
Are you sure? On my local machine this is a directory.